### ISSUE TITLE
Paella Player: Resolution management

### ISSUE THREAD

**Issue by oas777** [elan-ev/tobira#1127]
I was about to bash Berlin for recording in 4:3 when I saw the actual composition in the ETH portal:
![grafik](https://github.com/elan-ev/tobira/assets/1752342/da721a67-2ab9-4e61-ac25-2b5fb154199d)
vs.
![grafik](https://github.com/elan-ev/tobira/assets/1752342/9dccacae-664c-49eb-af57-0f8e5ddb2709)

https://tobira.opencast.org/v/AVoPw0z5uQ0 vs. https://video.ethz.ch/events/opencast/2023/berlin/69fa69d5-d976-4e22-9b82-0460d3fa7e04.html

**Comment by LukasKalbertodt** [comment]
I cropped the presenter before uploading it to the Opencast behind tobira.opencast.org. So this has nothing to do with Paella actually, the video is just different. I cropped it because 16:9 is pretty useless for that. I never move out of the 4.3 frame and when making the presentation video "larger", I find the composition to be very optimal (most space taken up by the important content, but the presenter is still visible):

![image](https://github.com/elan-ev/tobira/assets/7419664/62835ed2-ff11-46b5-b395-8d598c28c959)

So I guess this issue can be closed? Or am I misunderstanding you?
